cmd/utils/utils_test.go

```diff
@@ -0,0 +1,15 @@
+package utils_test
+
+import (
+	"errors"
+
+	"github.com/PennTex/commuter/cmd/utils"
+)
+
+// cant test this because process error exists the process
+func ExampleProcessError() {
+	err := errors.New("Test")
+	es := "Error String"
+
+	utils.ProcessError(err, es)
```

Comment:
Should we make the utils tests more robust/in line with the other testing standards?